Website/DesktopModules/Hotcakes/Core/Areas/Account/Controllers/WishListController.cs

```diff
@@ -112,7 +112,7 @@ private List<SavedItemViewModel> PrepItems(List<WishListItem> items)
             foreach (var item in items)
             {
                 var p = HccApp.CatalogServices.Products.FindWithCache(item.ProductId);
-
+                item.ProductShortDescription = p.Options.CartDescription(item.SelectionData.OptionSelectionList);
```

Comment:
@hismightiness We checked CartDescription at that time, There is condition to check if option is null then return string.empty. That was the reason we did not put condition p.Hasoption(). I also agree that we should add this condition so that in future if any changes will come in to the core method CartDescription we are fine with this area. I will put condition and generate new pull request.